internal/metrics/metrics.go

```diff
@@ -202,7 +203,10 @@ func (c *diskCollector) Describe(ch chan<- *prometheus.Desc) {
 }
 
 func (c *diskCollector) Collect(ch chan<- prometheus.Metric) {
-	usage := du.NewDiskUsage(c.path)
+	usage, err := du.New(c.path)
+	if err != nil {
+		log15.Error("error getting disk usage info", err)
```

Comment:
```suggestion
		log15.Error("error getting disk usage info", err)
		return
```

This is to prevent nil panics since `usage` will be `nil`.